CONTRIBUTING.rst

```diff
@@ -2,21 +2,20 @@ Contributing
 ============
 
 Contributions and issues are most welcome! All issues and pull requests are
-handled through github on the `dls_controls repository`_. Also, please check for
-any existing issues before filing a new one. If you have a great idea but it
-involves big changes, please file a ticket before making a pull request! We
-want to make sure you don't spend your time coding something that might not fit
-the scope of the project.
+handled through GitHub_. Also, please check for any existing issues before
+filing a new one. If you have a great idea but it involves big changes, please
+file a ticket before making a pull request! We want to make sure you don't spend
+your time coding something that might not fit the scope of the project.
 
-.. _dls_controls repository: https://github.com/dls-controls/dls_pmaccontrol/issues
+.. _GitHub: https://github.com/dls-controls/dls_pmaccontrol/issues
 
 Running the tests
 -----------------
 
 To get the source source code and run the unit tests, run::
 
     $ git clone git://github.com/dls-controls/dls_pmaccontrol.git
```

Comment:
URL needs updating too

Reply:
Thanks Gary - I've fixed that but I also need to update the docs on how to install and contribute as they still refer to pipenv (and need updating to include container dev)

Reply:
resolving for this project as I'm going to do the docs in ibek